InternetStoreCHIReloaded/BLL/Models/ProductsSet.cs

```diff
@@ -0,0 +1,18 @@
+﻿using DAL.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BLL.Models
+{
+    public class ProductsSet : IHasId
```

Comment:
What is ProductsSet? Is it part of the cart? Name is not obvious. please, rename it

Reply:
done